### DISCUSSION TITLE
Provided Google Play service account credentials could not be used: invalid_grant: Invalid grant: account not found

### DISCUSSION THREAD

**Discussion by Biomanuel** [codemagic-ci-cd/codemagic-docs#2674] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Linked Google Play Console to Cloud Project
2. Created Service account on Google Cloud console following the documentation
3. Created Service account API keys and downloaded JSON
4. Invited Service account email to Google Play Console
5. Made a global variable with the API Key Json content and named it GCLOUD_SERVICE_ACCOUNT_CREDENTIALS according to the documentation
6. Followed and Copied commands in the workflow into app settings
7. Then ran the build and publish to Play Store workflow.

### Expected results

Successful build and publishing to Play Store console.

### Actual results

Provided Google Play service account credentials could not be used: invalid_grant: Invalid grant: account not found

### Build id (optional)

66b378433f7585b508d0755a

**Comment by himesh-cm** [comment]
Hey @Biomanuel 
The error  `Provided Google Play service account credentials could not be used: invalid_grant: Invalid grant: account not found` occurs when there is an issue with your service json key, please make sure you have followed all the steps shown [here](https://docs.codemagic.io/yaml-publishing/google-play/#configure-google-play-api-access) in the docs. if you are unsure, please create a new key and try again.

**Reply by Biomanuel** [reply]
I have done everything in the docs, but I still get the error

**Reply by himesh-cm** [reply]
Did you create a new key with all the steps?

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
